### PR TITLE
Modify scanner to trash registers after call

### PR DESCRIPTION
**Pull request by ptomin**
- Create block work item unit test to verify trashed registers after call

### CODE REVIEW

File: subjects/regressions/reko-90/PP.c [hunk 1]
```diff
@@ -15021,7 +15020,7 @@ selector fn0800_A4F6(word16 bp, selector ds, word16 wArg02, word16 wArg04, word1
 word16 fn0800_A53C(byte al, selector ds, segptr32 ptrArg02, byte bArg06, word16 wArg08)
 {
 	word16 ax_24;
-	msdos_unknown_2143();
+	__syscall(33);
```
Comment (ptomin) [inline subjects/regressions/reko-90/PP.c]: This is regression. Is not it, John?

Comment (uxmal) [reply subjects/regressions/reko-90/PP.c]: Yes. We should still be generating `msdos_unknown_2143` here.

Comment (ptomin) [reply subjects/regressions/reko-90/PP.c]: Fixed in 85dbdeb

**Comment by uxmal** [comment]
Looks good.

**Pull request merged**